docs/guides/security-guide.md

```diff
@@ -132,18 +119,23 @@ by the ca.crt (can copy the generated files from the server if client is not
 on the same system as server).
 
 ### P4RT client
-
-The p4rt-ctl (P4RT client) will default to communicate via secure mode
-(port 9559). If certificates are not available, the P4RT client will attempt a
+The P4Runtime Control client will default to communicate via secure mode using
+port 9559. If certificates are not available, the P4RT client will attempt a
 connection using insecure client credentials as a fallback mechanism. Note that
-the communication may fail if infrap4d runs in secure mode only.
+the communication will fail if infrap4d runs in secure mode only. Both server and
```

Comment:
Do we support a mode in which the _server_ falls back from secure mode to insecure mode? That's what "runs in secure mode _only_" implies. A more correct wording would be:

```suggestion
the communication will fail if only infrap4d runs in secure mode. Both server and
```